src/applications/my-education-benefits/tests/fixtures/data/claimantInfoResponse.js

```diff
@@ -10,13 +10,13 @@ export const claimantResponse = {
         firstName: 'Teressa',
         lastName: 'Harber',
         middleName: 'Estefana',
-        notificationMethod: 'TEXT',
+        notificationMethod: 'text',
         contactInfo: {
           addressLine1: '23082 Xavier Union',
           addressLine2: 'Apt. 498',
           city: 'Lake Stephanville',
           zipcode: '40638-9651',
-          emailAddress: null,
+          emailAddress: 'fake@fake.com',
```

Comment:
Hi @department-of-veterans-affairs/my-education-benefits 👋. I'm not sure why some of your test data needed to be changed but I couldn't get this PR to pass without making some updates in a few places. Let me know if this is already something you're working on or if these changes look ok to you.

Reply:
And I should mention that the failing tests seem to be unrelated to what this PR is trying to do (update the component-library to the latest version).

Reply:
HI @jamigibbs, the changes look fine. We also had some issues with the CI because of these e2e tests, they haven't been well-maintained or updated recently. We folded similar changes into a feature branch struggling through the CI checks. And have an enabler for updating the e2e tests for these products to be worked on in more detail in the future. Feel welcome to merge this in.